crates/adapter/src/fastly/core.rs

```diff
@@ -268,6 +268,24 @@ pub mod fastly_abi {
     }
 }
 
+pub mod fastly_vcpu {
```

Comment:
(I can imagine a lot of additional introspection we might want to add over time, on other aspects of the guest behavior.)

Reply:
I went with `fastly_compute_runtime`, which I like as a general future module for getting or setting or otherwise looking at the behavior of a session itself.